skeleton/webpack/aurelia_project/tasks/build.ext

```diff
@@ -1,67 +1,8 @@
-// @if feat.babel
-import webpackConfig from '../../webpack.config';
-import webpack from 'webpack';
-import project from '../aurelia.json';
-import gulp from 'gulp';
-import del from 'del';
-// @endif
-// @if feat.typescript
-import * as webpackConfig from '../../webpack.config';
-import * as webpack from 'webpack';
-import * as project from '../aurelia.json';
-import * as gulp from 'gulp';
-import * as del from 'del';
-// @endif
-import {CLIOptions, Configuration} from 'aurelia-cli';
-import configureEnvironment from './environment';
+import { NPM } from 'aurelia-cli/lib/package-managers/npm';
 
-const analyze = CLIOptions.hasFlag('analyze');
-const buildOptions = new Configuration(project.build.options);
-const production = CLIOptions.getEnvironment() === 'prod';
-const server = buildOptions.isApplicable('server');
-const extractCss = buildOptions.isApplicable('extractCss');
-const coverage = buildOptions.isApplicable('coverage');
+export default function() {
+  console.log('`au build` is an alias of the `npm run build`, you may use either of those; see README for more details.');
 
-const config = webpackConfig({
-  production, server, extractCss, coverage, analyze
-});
-const compiler = webpack(/* @if feat.typescript **<any>/* @endif */config);
-
-function buildWebpack(done) {
-  if (CLIOptions.hasFlag('watch')) {
-    compiler.watch({}, onBuild);
-  } else {
-    compiler.run(onBuild);
-    compiler.hooks.done.tap('done', () => done());
-  }
-}
-
-function onBuild(err, stats) {
-  if (!CLIOptions.hasFlag('watch') && err) {
-    console.error(err.stack || err);
-    if (err.details) console.error(err.details);
-    process.exit(1);
-  } else {
-    process.stdout.write(stats.toString({ colors: require('supports-color') }) + '\n');
-
-    if (!CLIOptions.hasFlag('watch') && stats.hasErrors()) {
-      process.exit(1);
-    }
-  }
+  const args = process.argv.slice(3);
+  return (new NPM()).install(['--', ...args], process.cwd(), 'run');
```

Comment:
Did you test this? I think you missed `'build'`.
```js
return (new NPM()).install(['build', '--', ...args], process.cwd(), 'run');
```

Reply:
Sorry, I missed it. I did test the changes. Actually, just realized that there are no release-check in place for `build` for webpack skeletons.

Reply:
Oh, I see. I will add more check in release-check.

Reply:
@3cp I have made the changes including the one you pointed above. Also added release-check for `au build` for `webpack` skeleton. I am running the release checks now. Will update this PR once it is successful

Reply:
done